packages/host/app/components/card-pill.gts

```diff
@@ -86,6 +106,73 @@ export default class CardPill extends Component<CardPillSignature> {
       .is-autoattached {
         border-style: dashed;
       }
+      .card-content {
+        display: flex;
+        max-width: 100px;
+      }
+      :deep(.atom-format) {
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+      }
+    </style>
+  </template>
+}
+
+export default class CardPill extends Component<CardPillSignature> {
+  @tracked _isTooltipDisplayed = false;
+
+  get component() {
+    return this.args.card.constructor.getComponent(this.args.card, 'atom');
+  }
+
+  get isTooltipDisplayed() {
+    // If card is an auto-attached card, we need to hide the tooltip
+    // because there is another tooltip shows "Topmost card is shared automatically".
+    return this._isTooltipDisplayed && !this.args.isAutoAttachedCard;
```

Comment:
1. There are now 2 clashing tooltips - how do you decide which one is more important?
2. Logic got confusing because the pill component now has more context about its surroundings, for example needing to know there is another tooltip... Ideally you would just pass a tooltip into here and display it if it's present

Have you considered what Buck suggested, a `title` tag?

<img width="293" alt="image" src="https://github.com/cardstack/boxel/assets/273660/fa19fa78-8cac-48ca-9cc3-4ca163e94e22">

Reply:
Thanks for the feedback! I didn't realize about the title attribute before. I reverted the commit regarding the tooltip and implemented title attribute instead. Please check it again.

Reply:
Instead of adding title in `field-component` I moved it to `card-pill` in this [PR](https://github.com/cardstack/boxel/pull/1147) so we don't need to update `field-component` and keep any changes in `card-pill` file.